apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -154,7 +162,7 @@ function load_posts_list_block() {
  */
 function load_starter_page_templates() {
 	// We don't want the user to choose a template when copying a post.
-	// phpcs:ignore WordPress.Security.NonceVerification.NoNonceVerification
+	// phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Nothing to do here, but wanted to just point out this must be the thousandth time we bounce this back and forth, and I guess our PHPCS configs are misaligned.

Your rules throw a warning for `.Recommended`, while mine for `.NoNonceVerification`. 🤷‍♂ 

<img width="835" alt="Screenshot 2019-11-13 at 15 26 28" src="https://user-images.githubusercontent.com/2070010/68777789-20636780-062a-11ea-92c1-3d0bf5296582.png">

Reply:
Ack yeah. I don't know how they would be different... Maybe I'll try updating my phpcs config somehow?